aiida/aiida_config.yaml

```diff
@@ -0,0 +1,21 @@
+store_path: /work/playground2/repo
```

Comment:
would it make sense to write this file on project startup (e.g. in the post-init script) so you could set the values for e.g. email, name correctly for the user?

Reply:
yes.

I originally thought of simply removing the user-level variables from the `aiida_config.yaml` and have the command read them from the environment but we can also simply create the `aiida_config.yaml` outside the git repo. 
will do!

Reply:
ah ok it looked to me like you can't read this from the environment - but if that's an option it might be better? Trouble is that either way the project won't run properly outside of renkulab. With the environment variables maybe at least you would have a chance to set them in a local environment and the project can still work.